site/layouts/shortcodes/orange-supra.html

```diff
@@ -10,6 +14,44 @@
       <li class="nav-item"><a href="#" class="nav-link active" aria-current="page">Personal</a></li>
       <li class="nav-item"><a href="#" class="nav-link">Business</a></li>
     </ul>
-    {{- $input -}}
+{{- if eq $mode "languages" }}
+    <ul class="navbar-nav ml-auto">
```

Comment:
I wonder why there is .ml-auto as this class doesn't exist anymore. The new one is .ms-auto + it works well without it.

Reply:
You're totally right, I remove the useless `.ml-auto` class for supra bars.